internal/operands/common-templates/reconcile_test.go

```diff
@@ -197,6 +197,20 @@ var _ = Describe("Common-Templates operand", func() {
 			Expect(updatedTpl.Labels[TemplateTypeLabel]).To(Equal("base"), TemplateTypeLabel+" should equal base")
 			Expect(updatedTpl.Labels[TemplateVersionLabel]).To(Equal("not-latest"), TemplateVersionLabel+" should equal not-latest")
 		})
+
+		It("should add deprecated label in old templates", func() {
```

Comment:
Can you merge this test with the previous one? Just add the label check at the end and change the test's description.